1-Authentication/2-sign-in-b2c/SPA/src/app/auth-config.ts

```diff
@@ -59,11 +66,22 @@ export const b2cPolicies = {
     }
 }
 
+/**
+ * Scopes you add here will be prompted for user consent during sign-in.
+ * By default, MSAL.js will add OIDC scopes (openid, profile) to any login request.
+ * For more information about OIDC scopes, visit:
+ * https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#openid-connect-scopes
+ */
+export const loginRequest = {
+    scopes: []
+}
+
 /**
  * An optional silentRequest object can be used to achieve silent SSO
- * between applications by providing a "login_hint" property.
+ * between applications by providing a "loginHint" property. For more, visit:
+ * https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-js-sso#sso-between-different-apps
```

Comment:
Add loginHint steps in the readme

Reply:
Good point -will need to check if this applies to B2C :/

Reply:
added link to show how to enable to username in id token (which can be used as loginHint later)